src/picongpu/submit/pizdaint-cscs/picongpu.profile.example

```diff
@@ -0,0 +1,56 @@
+# this file is loaded from all PIConGPU template files `*.tpl` therefore please
+# copy this file to `$SCRATCH/picongpu.profile`
+#
+
+# General modules #############################################################
+#
+module swap PrgEnv-cray PrgEnv-gnu
+module load cmake/3.3.2
+module load cudatoolkit/7.0.28-1.0502.10742.5.1
+
+# Libraries ###################################################################
+module load boost
```

Comment:
I would feel more save if you pin this to a specific, latest version of boost.

sometimes jobs are in the queue for very long and the default might change.